include/wlr/types/wlr_keyboard.h

```diff
@@ -47,6 +47,8 @@ struct wlr_keyboard_modifiers {
 	xkb_mod_mask_t group;
 };
 
+struct wlr_virtual_keyboard_v1;
```

Comment:
This forward declaration is unnecessary.

Reply:
Removed, done.